ts/src/utils/rpc.ts

```diff
@@ -82,7 +82,7 @@ async function getMultipleAccountsCore(
   if (commitment) {
     args.push({ commitment });
   }
-  // @ts-expect-error
+  // @ts-ignore
```

Comment:
This method should be part of web3.js at some point in the future, if not already.